drs_downloader/download.py

```diff
@@ -101,7 +102,7 @@ class DownloadURL(object):
 
 async def _download(urls: List[DownloadURL], dest: Path):
```

Comment:
The data class DownloadURL should be renamed to something that describes its contents.
See [DrsObject](https://ga4gh.github.io/data-repository-service-schemas/preview/release/drs-1.0.0/docs/#_drsobject)

Reply:
Line 86: `assert expected_md5 == actual_md5, f"Actual md5 {actual_md5} does not match expected {expected_md5}"` should not be uncommented

Reply:
There was a bug in the initial downloader.py


```def parts_generator(size, start=0, part_size=1 * 512 ** 2):
    while size - start > part_size:
        yield start, start + part_size
        start += part_size + 1
    yield start, size
```